GetCertificate/handler.ts

```diff
@@ -102,7 +104,55 @@ export const GetCertificateHandler = (
     TE.chain(e => {
       switch (e.status) {
         case 200:
-          return TE.of(e.value);
+        case 404:
+          return pipe(
+            e,
+            TE.fromPredicate(
+              i => i.status === 200,
+              () => ({ info: " ", status: ExpiredEnum.expired })
```

Comment:
I think you're using `" "` to avoid NonEmptyString parsing fail, right?

If so, can we use a placeholder string instead? 
Rationale: `" "` is ambiguos, someone may consider it empty. We may consider to trim values before validation.

Reply:
This is a workaround: our business team required us to not show anything and the blank space used is a unicode "HAIR SPACE" tested by App team to not broke the markdown rendering.
This is a sub-optimal solution in order to not havt to release a new app version.

Reply:
Moved on a commented constant.